libs/SalesforceSDK/build.gradle

```diff
@@ -11,34 +11,22 @@ dependencies {
     api project(':libs:SalesforceAnalytics')
     api 'com.squareup.okhttp3:okhttp:3.12.12'
     api 'com.google.firebase:firebase-messaging:20.1.0'
```

Comment:
The reasons we haven't updated `okhttp` and `firebase-messagine` to the latest are unrelated from API version, correct?

Reply:
Yes, they're unrelated, and we shouldn't bump them up yet.